independent-publisher-connectors/DPIRD Science/Readme.md

```diff
@@ -0,0 +1,67 @@
+
+## DPIRD Science Connector for Western Australia
+This API provides the ‘back end’ to the [Rainfall to Date](https://www.agric.wa.gov.au/climate-weather/rainfall-date-tool-western-australia/), [Potential Yield](https://www.agric.wa.gov.au/climate-weather/potential-yield-tool/), and [Soil Water](https://www.agric.wa.gov.au/climate-weather/soil-water-tool/#?station=10115&soil=3&groundCover=1) tools already available on the DPIRD website. These online climate forecasting tools assist grain growers to optimise their crop potential.
+
+* The [Soil Water tool](https://www.agric.wa.gov.au/climate-weather/soil-water-tool/#?station=10115&soil=3&groundCover=1) models how rainfall moves through the layers of different soil types to identify how much water is available for plant roots to take up.
+* The [Rainfall to Date](https://www.agric.wa.gov.au/climate-weather/rainfall-date-tool-western-australia/) tool allow growers to select the beginning and end of the season to better determine available soil water for crop use.
+* The [Potential Yield calculator](https://www.agric.wa.gov.au/climate-weather/potential-yield-tool/) combines weather data from the department and Bureau of Meteorology’s weather stations with historical decile finishes to produce the maximum yield crop potential.
+
+You can see what information is available from the API, and how you can filter and search for specific information, by exploring the [interactive home page](https://www.agric.wa.gov.au/science-api-20).
+
+## Prerequisites
+You will need the following to proceed
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+* The Power platform CLI tools
+* An [API key](https://www.agric.wa.gov.au/form/dpird-api-registration) from DPIRD in accordance with [terms and conditions](https://www.agric.wa.gov.au/apis/api-terms-and-conditions)
+
+## Authentication

```

Comment:
There is no Obtaining credentials section which is a required section in the readme file. You can always refer to this [template ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)for more information on the readme file.